console/src/main/scala/io/shiftleft/console/Console.scala

```diff
@@ -224,8 +224,13 @@ class Console[T <: Project](executor: AmmoniteExecutor, loader: WorkspaceLoader[
   def undo: File = {
     project.overlayDirs.lastOption
       .map { dir =>
-        val zipFiles = dir.list.toList.sortWith(compareFiles).reverse.map(_.path.toString)
-        CpgLoader.addDiffGraphs(zipFiles, cpg)
+        if (dir.isRegularFile) {
+          System.err.println("Detected undo information in old format. That's ok.")
```

Comment:
`System.err` doesn't sound "ok"?

Reply:
We use System.err in Ocular to report things back to the user. We do that because Ocular may also run in interactive scripts and then these messages don't end up in stdout.

Reply:
If that's the norm, then ok. I would really like to see us using some logger here instead.